### PR TITLE
Form HTML and CSS

### PR DESCRIPTION
**Pull request by Juank628**
## First code review: Form HTML and CSS
In this pull request I:
- Added index.html
- Added style.css
- Complete the code for the form (HTML and CSS)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.title-container {
+    text-align: center;
```
Comment (Juank628) [inline css/style.css:2]: For readability please use 2 spaces instead tab key for indentation 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+      
+      <div  class="option-column">
+        <label for="name" id="name-label">Name</label>
+        <input placeholder="Your Name" required id="name" type="text">
+        <label for="email" id="email-label">Email</label>
+        <input placeholder="Your Email" required id="email" type="email">
+        <label for="number" id="number-label">Number</label>
+        <input max="10" min="2" placeholder="Your Number" id="number" type="number">
+      </div>
```
Comment (Juank628) [inline index.html:25]: Please try to use a "fieldset" tag instead a general "div". That makes more readable for web browsers 🔢

**Review by Juank628 — COMMENTED** [review]
## Changes required 🔢 
Good job!!! I only have some comments to make it even better 🥇 . Please check my comments below.
Submit for a re-review when you have done all the changes 👍

**Review by aimalamiri — APPROVED** [review]
Hi @Juank628, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- Good job for using the  GitHub review comments ✔️ ⭐
- The comments are descriptive and respective 💯
- Good use of the emojis 🗃️


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**